inc/saiport.h

```diff
@@ -1214,6 +1217,15 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PTP_MODE,
 
+    /**
+     * @brief Serdes object ID for the port
+     *
+     * @type sai_object_id_t
+     * @flags READ_ONLY
+     * @objects SAI_OBJECT_TYPE_PORT_SERDES
+     */
+    SAI_PORT_ATTR_PORT_SERDES_ID,
```

Comment:
usually a port has a number of serdeses, why this not a list of serdes objects and each serdes has its attributes to control.

Reply:
by default, does the SAI create the port serdes object?

Reply:
Yes. SAI needs to populate serdes object